src/modules/Tasks.js

```diff
@@ -0,0 +1,7 @@
+export default class Tasks {
+  constructor(index, description, completed) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
```

Comment:
I suggest to give a default value for the functions parameters, what will happen if there is no parametes passed to the function?:no_mouth: